src/catalog/src/builtin.rs

```diff
@@ -3800,7 +3879,8 @@ FROM
 pub const MZ_DATAFLOW_OPERATOR_PARENTS_PER_WORKER: BuiltinView = BuiltinView {
     name: "mz_dataflow_operator_parents_per_worker",
     schema: MZ_INTERNAL_SCHEMA,
-    sql: "CREATE VIEW mz_internal.mz_operator_parents_per_worker AS
```

Comment:
The name differs here. `mz_dataflow_operator_parents_per_worker` vs. `mz_operator_parents_per_worker`.